org.eclipse.m2e.apt.core/lifecycle-mapping-metadata.xml

```diff
@@ -96,6 +96,19 @@
         </configurator>
       </action>
     </pluginExecution>
+    <pluginExecution>
+      <pluginExecutionFilter>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-toolchains-plugin</artifactId>
+        <versionRange>[2.0,)</versionRange>
+        <goals>
+          <goal>toolchain</goal>
+        </goals>
+      </pluginExecutionFilter>
+      <action>
+        <ignore></ignore>
+      </action>
+    </pluginExecution>
```

Comment:
Since this the maven-toolchains-plugin is a very general one I think its silencing is better placed into `org.eclipse.m2e.core/lifecycle-mapping-metadata.xml`

Reply:
I've moved the metadata. Thank you for the hint,